core/arch/arm/kernel/kern.ld.S

```diff
@@ -50,13 +50,19 @@
 
 #include <platform_config.h>
 
+#ifndef SMALL_PAGE_SIZE
+#define SMALL_PAGE_SIZE		4096
+#endif
+
 OUTPUT_FORMAT(CFG_KERN_LINKER_FORMAT)
 OUTPUT_ARCH(CFG_KERN_LINKER_ARCH)
 
 ENTRY(_start)
 SECTIONS
 {
 	. = CFG_TEE_LOAD_ADDR;
+	. = ALIGN(SMALL_PAGE_SIZE);
+	__core_unpg_rx_start = .;
```

Comment:
The "core" part isn't needed and also inconsistent with all the other symbols defined here.
Isn't `__core_unpg_rx_start` supposed to be exactly the same thing as `__text_start` or am I missing something?